addon/-infra/potential-packages.js

```diff
@@ -1,18 +0,0 @@
-/**
```

Comment:
seems odd to be deleting this as part of this PR

Reply:
checking the CI you're right this is why the 3.12 tests are failing